### ISSUE TITLE
Friendz for Mac feature requests / bugs

### ISSUE THREAD

**Issue by Shawzborne** [thermogl/TIXboxLiveEngine#1]
**Bugs:**

-the app locks up when you open a macbook air from sleep, everything is blank and you have to quit and relaunch the app. this is on mavericks. 2013 top line mac air. I frequently open and close my air (who doesn't)

**Major improvements**

**Push Notifications:**
-Quick Reply
-See who the xbox live message is from
-Message 3 line preview so you dont have to click on the menu bar to see it and you can just quick reply to the message
-Xbox Achievement sound for the push notification

**GUI**

-Plus + icon to be able to add Xbox Live gamertags and specifiy which list to add the gamertag to.

Make the plus sign dual purpose make it allow you to compose a message as well as be able to use to add gamertags

-Favourite gamertags support, be able to add and remove gamertags from favourites list on xbox live account 

-M2AF Message to all friends ability, Message to all Favourites ability use 360Live iphone app as example

-Move Messages tab to centre and the star (achievos to the outside)

-GUI Refresh make it black transparent like ios 7 keyboard maybe? 

-Inbox (Messages) view show the gamerpicture on the left beside their name

-Edit Xbox Live Bio, motto, Location/Country etc directly from the app (coolbeans)
to do this you would tap your gamer pic/name and would open the settings etc

-Direct link to a Xbox One Gamerpic 1080p http://avatar.xboxlive.com/avatar/e/avatarpic-xl.png
I suggest maybe being able to mouse over their pic and see the full view(would be sexy)
- Be able to view a gamertag profile with bio etc without opening a web browser 
- Be able to compare games without a browser

-Add search so as you type gamertags pop up 

**Minor Improvements**

-Mavericks OSX 10.9 performance improvements sdk additons etc.

In like 2016:
-Be able to add xbox one pins etc

**Comment by thermogl** [comment]
All fine suggestions.

One point: Searching every gamertag in existense wouldn't be very performant for auto-complete. Friends, recent players and message senders is enough, I think.

**Comment by Shawzborne** [comment]
ok I will remove that one, if you want me to try to draw what the interface could look like let me know Im not very good at drawing either but I already have what it would look like in my mind specially make it black instead of old school mac look

**Comment by Shawzborne** [comment]
so non gey interface make is sexy glossy/transparent black like the xbox one

**Comment by Shawzborne** [comment]
I hope you can maybe do the push notification soun and quick reply and preview items first that is the most important for me anyway

**Comment by thermogl** [comment]
During a trip down memory lane I just rediscovered your comments and I want to reiterate how much I appreciated your feedback. People like you are the reason I made the app available as a download. Initially it was just for me, but I figured maybe other people would find it useful. Obviously MSFT have an official app now, but from time to time I still think about what Friendz could be today. Again, thank you so much for your input and I hope you’re doing well. 

-Tom
